src/ui/public/timepicker/kbn_global_timepicker.js

```diff
@@ -2,6 +2,7 @@ import moment from 'moment';
 import UiModules from 'ui/modules';
 import chromeNavControlsRegistry from 'ui/registry/chrome_nav_controls';
```

Comment:
this import is obsolete.  linter was also complaining about unused function arguments in this file as well: `config`, `newval`, `oldVal`, `$el`, `attrs`

While we're at it, we may as well remove them ;)